bin/lib/common-functions.sh

```diff
@@ -21,6 +21,8 @@ safe_colors() {
 }
 
 setup_trace() {
+    # set TRACE if GHA debug logging is enabled
+    [[ ${RUNNER_DEBUG:-0} -eq 1 ]] && TRACE=1
```

Comment:
This will allow us to enable debug logging when we re-run a job.

<img width="341" height="206" alt="Image" src="https://github.com/user-attachments/assets/12ea7691-350b-4fa1-a522-29bb8525122c" />